src/apis/PyPI.js

```diff
@@ -0,0 +1,54 @@
+const { APIWrapper } = require('../')
+const axios = require('axios')
+const cheerio = require('cheerio')
+
+const API_URL = 'https://pypi.org/'
+
+module.exports = class PyPI extends APIWrapper {
+  constructor () {
+    super({
+      name: 'pypiapi'
+    })
+  }
+
+  async search (query) {
+    const html = await axios({
+      url: `${API_URL}search/?q=${query}`
+    })
+    const parsed = cheerio.load(html.data)
```

Comment:
I feel like having the scraper directly in the bot is not a very good idea. I believe it should stay on its own, like the MDN documentation https://github.com/SwitchbladeBot/mdn-doc using Cloudflare workers